.github/workflows/openbsd.yaml

```diff
@@ -0,0 +1,66 @@
+name: Staged commit - OpenBSD
+
+on:
+  push:
+    branches: [ "bsd-merge-tests" ]
+
+env:
+  # empty except for pull_request events
+  PULL_REQUEST_NUMBER: ${{ github.event.pull_request.number }}
+
+jobs:
+
+  openbsd:
+    runs-on: ubuntu-latest
+    if: false
+
+    steps:
+      - name: Checkout Squid sources
+        uses: actions/checkout@v4
+
+      - name: Test
+        id: test
+        uses: vmactions/openbsd-vm@v1
+        with:
+          usesh: true
+          prepare: |
+            pkg_add \
+              autoconf-2.72p0 \
+              automake-1.16.5 \
+              bash \
+              coreutils \
+              cppunit \
+              git \
+              ggrep \
+              gmake \
+              libxml \
+              libtool \
+              m4 \
+              metaauto
+
+            mkdir $HOME/bin
+            ln -s /usr/local/bin/ggrep $HOME/bin/grep
+            ls -ld /usr/local/share/aclocal*
+            ls -l /usr/local/bin/auto*
+
+          run: |
+            export MAKE=gmake
+            export pjobs="-j`gnproc`"
+            export AUTOMAKE_VERSION=1.16
+            export amver=${AUTOMAKE_VERSION}
+            export ACLOCAL_AUTOMAKE_DIR="/usr/local/share/aclocal-${AUTOMAKE_VERSION}"
+            export ACLOCAL_PATH="/usr/local/share/aclocal:/usr/local/share/aclocal-${AUTOMAKE_VERSION}"
```

Comment:
What happens if we do not define these two variables? Is there a simple way to avoid this and similar dependencies between "pgk_add" adding X above and Squid build commands using X below?

Reply:
Obsolete PR version, but to make a long story short, I defined them because otherwise the build fails.
OpenBSD's autotools setup is weird